packages/near-membrane-base/src/blue.ts

```diff
@@ -125,8 +106,19 @@ export function blueProxyFactory(env: MembraneBroker) {
         ObjectDefineProperties(shadowTarget, blueDescriptors);
     }
 
-    function createBlueProxy(red: BlueProxyTarget): BlueProxy {
-        const shadowTarget = createBlueShadowTarget(red);
+    function createBlueProxy(
```

Comment:
now, anyone trying to create a proxy, whether it is blue or red, should provide the anchor (target -> in the future remote reference) and the shape of the shadow target to be created.